src/common/dto/users/request/update.user.request.ts

```
@@ -2,6 +2,8 @@ import { ApiProperty } from '@nestjs/swagger';
 
 import { IsString, IsUrl, IsOptional, IsBoolean, IsInt } from 'class-validator';
 
+import { IsString, IsUrl, IsOptional, IsBoolean, IsInt } from 'class-validator';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'IsString'. Consider to delete it or rename it.

'IsString' is defined here:


(lint/suspicious/noRedeclare)